pyart/retrieve/cfad.py

```diff
@@ -0,0 +1,93 @@
+"""
+Create CFAD from a radar or grid field
+
+"""
+
+import numpy as np
+
+
+def create_cfad(
+    radar,
+    field_bins,
+    altitude_bins,
+    field="reflectivity",
+    field_mask=None,
+    min_frac_thres=0.1,
+):
+    """
+    This function returns a Contoured Frequency by Altitude Diagram (CFAD; Yuter et al. 1995), a 2-dimensional
+    histogram that is normalized by the number of points at each altitude. Altitude bins are masked where the counts
+    are less than a minimum fraction of the largest number of counts for any altitude row.
+
+    radar : Radar
+        Radar object used. Can be Radar or Grid object.
+    field_bins : list
+        List of bin edges for field values to use for CFAD creation.
+    altitude_bins : list
+        List of bin edges for height values to use for CFAD creation.
+    field : str
+        Field name to use to look up reflectivity data. In the
+        radar object. Default field name is 'reflectivity'.
+    field_mask : array
+        An array the same size as the field array used to mask values.
+    min_frac_thres : float, optional
+        Fraction of values to remove in CFAD normalization (default 0.1). If an altitude row has a total count that
+        is less than min_frac_thres of the largest number of total counts for any altitude row, the bins in that
+        altitude row are masked.
```

Comment:
Space line between the headers of each section, Returns, references etc I think is needed for sphinx to generate correctly.